### ISSUE TITLE
Java 17 Build is broken due to Record

### ISSUE THREAD

**Issue by meltsufin** [googleapis/java-pubsublite-flink#82]
```
/home/runner/work/java-pubsublite-flink/java-pubsublite-flink/src/main/java/com/google/cloud/pubsublite/flink/PubsubLiteSourceSettings.java:[171,23] error: reference to Record is ambiguous
```

**Comment by chanseokoh** [comment]
@anguillanneuf friendly ping

**Comment by anguillanneuf** [comment]
@meltsufin and @chanseokoh - this library is still in development. I removed the P2 tag.

**Comment by anguillanneuf** [comment]
Fixed by #86
